frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -58,10 +62,10 @@ export async function request(
   const id = setTimeout(() => controller.abort(), timeout);
 
   const url = combinePath(BASE_HTTP_URL, fullPath);
+  const requestData = { signal: controller.signal, ...opts };
   let response: Response = new Response(undefined, { status: 502, statusText: 'Unreachable' });
-
```

Comment:
Should not be here (unless prettier did this :stuck_out_tongue: ).

Reply:
I guess it must have removed the line.